data/obs-packaging/README.rst

```diff
@@ -0,0 +1,39 @@
+.. contents::
+.. sectnum::
+
+``Clear Containers OBS spec files``
+====================================
+
+
+The `Clear Containers`_ packages were made available via `Open Build Service`_ to allow interested parties the opportunity to try out the technology, even if they are not using Clear Linux directly.
+Once `cc-oci-runtime`_ or any of its components are updated we release a new package or set of packages.
+
+Repository:
+
+- http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/
+
+Components
+----------
+
+- `cc-oci-runtime`_
+- `clear-containers-image`_
+- `clear-containers-selinux`_
+- `linux-container`_
+- `qemu-lite`_
```

Comment:
I'd be tempted to add a brief description to each of these components. Something like:

- `clear-containers-image`_: Contains the mini-OS required to run Clear Containers.
- `linux-container`_: The optimised guest kernel required to run Clear Containers.
- `qemu-lite`_: An optimised version of the QEMU hypervisor.